docs/examples/e05_commands_framework/main.py

```diff
@@ -55,19 +60,29 @@ async def handle_owner_only_error(ctx, exception):
     return False
 
 
-# Annotated parameters are parsed if there is a converter for the type. The most common entities and generic types
-# have converters added.
+# Annotated parameters are parsed if there is a converter for the type.
 #
-# You can send response message by returning or yielding any value from a command.
+# You can send response message by returning or yielding any value from a command. These 3 commands do the same thing
+# (but are not identical!):
 @Sakuya.commands
-async def about_role(role: 'Role'):
-    """Returns the role's identifier."""
+async def about_role_return(role: 'Role'):
+    """Sends the passed role id message to the context channel."""
     return role.id
 
 
+@Sakuya.commands
+async def about_role_yield(role: 'Role'):
+    """Sends the passed role id message to the context channel."""
+    yield role.id
+
+
+@Sakuya.commands
+async def about_role_send(ctx, role: 'Role'):
+    """Sends the passed role id message to the context channel."""
+    await ctx.send(role.id)
```

Comment:
Same 3 commands, but with different topping on them. Would be better if each 3 would use different entity type, like channel, user, emoji, sticker, ..